src/output-logger.ts

```diff
@@ -86,11 +86,14 @@ export class OutputLogger implements AbstractOutputLogger {
  * @param args - Arguments to `stream.write`: either one or more messages, or a
  * format string followed by values.
  */
-export function logToStream(stream: SimpleWriteStream, args: [string, ...any]) {
-  const [messageOrFormatString, ...rest] = args;
-  if (rest.length > 0) {
-    stream.write(format(`${messageOrFormatString}\n`, ...rest));
+export function logToStream(stream: SimpleWriteStream, args: unknown[]) {
+  if (typeof args[0] === 'string' && /%[sdifjoOc%]/u.test(args[0])) {
```

Comment:
What does this regular expression do?

Reply:
This checks for the existence of a format specifier, which `util.format` takes. I've extracted this check and documented it to make it more clear: [8b53d27](https://github.com/MetaMask/module-lint/pull/33/commits/8b53d273733f9ff0a69d21145de141addd2be984)